README.md

```diff
@@ -56,30 +56,26 @@ ptime <- timeCurrent
 * Getting the current year:
 ```haskell
 -- With time
-import Data.Time.Clock
-import Data.Time.Calendar
+import Data.Time
 
-currentYear <- (\(y,_,_) -> y) . toGregorian . utcDay <$> getCurrentTime
+currentYear <- (\(y,_,_) -> y) . toGregorian . utctDay <$> getCurrentTime
 
 -- With hourglass
 import System.Hourglass
-import Data.Time
+import Data.Hourglass
 
 currentYear <- dateYear . timeGetDate <$> timeCurrent
 ```
 
 * Representating "4th May 1970 15:12:24"
 ```haskell
 -- With time
-import Data.Time.Clock
-import Date.Time.Calendar
+import Data.Time
 
-let day = fromGregorian 1970 5 4
-    diffTime = secondsToDiffTime (15 * 3600 + 12 * 60 + 24)
-in UTCTime day diffTime
+UTCTime (fromGregorian 1970 5 4) (timeOfDayToTime $ TimeOfDay 15 12 24)
```

Comment:
Actually, for getting a `UTCTime`, this is probably more idiomatic that using `timeOfDayToTime`):

```suggestion
localTimeToUTC utc $ LocalTime (fromGregorian 1970 5 4) (TimeOfDay 15 12 24)
```